demisto_sdk/commands/validate/tests/BC_validators_test.py

```diff
@@ -54,7 +54,8 @@
 
 
 # Create a new content item with 3 commands with unique names. all commands have only 1 argument except the third command which has 2 arguments.
-generic_integration_with_3_commands_and_4_args = create_integration_object(
+
+GENERIC_INTEGRATION_WITH_3_COMMANDS_AND_4_ARGS = create_integration_object(
```

Comment:
Why is the number of args & command important here? cant we call it just DUMMY_INTEGRATION?

Reply:
I think it helps to understand the test, especially when i an adding a new arg, so it is clear to know what did we have before and what we have now.